src/components/Paginate.tsx

```diff
@@ -33,7 +33,9 @@ export default function Paginate({
       {/* <!-- This example requires Tailwind CSS v2.0+ --> */}
       <div className="px-4 py-3 flex justify-between sm:px-6">
         <div className="flex-1 flex justify-between sm:hidden">
-          {currentPage !== 1 && (
+          {currentPage === 1 ? (
+            <span></span>
+          ) : (
```

Comment:
With this change, the pagination element's "Next" button should always appear on the righthand side of the screen, even while on the first page.

I think this creates a more consistent experience.

![Screen Shot 2020-12-23 at 9 43 10 AM](https://user-images.githubusercontent.com/43425812/103023681-46348e80-4503-11eb-8119-29de7ccf4e70.png)